database/seed.js

```diff
@@ -0,0 +1,74 @@
+var mongoose = require('mongoose');
+const faker = require('faker');
+const dateGenerator = require('random-date-generator');
+const moment = require('moment');
+
+mongoose.connect('mongodb://localhost/reviews', {useNewUrlParser: true});
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connected to db')
+});
+
+var Schema = mongoose.Schema;
+
+var reviewSchema = new Schema({
+  housingId: Number,
+  reviewId: Number,
+  name: String,
+  date: String,
+  displayDate: String,
+  comment: String,
+  imgURL:String,
+  ratings: {
+    communication: Number,
+    accuracy: Number,
+    location: Number,
+    checkIn: Number,
+    cleanliness: Number,
+    value: Number
+    }
+  });
+
+
+var Review = mongoose.model('Reviews', reviewSchema);
+
+var rating = () => {
+  var ratings = [1, 2, 3, 4, 5];
+
+  return ratings[Math.floor(Math.random() * ratings.length)];
+}
+
+var save = () => {
+  for (var i = 0; i < 10; i++) {
+    const filter = {reviewId: i};
+    const update = {
+      housingId: 12345,
+      reviewId: i,
+      name: faker.fake("{{name.firstName}}"),
+      date: 'July 2015',
```

Comment:
Is it intentional that 'date' and 'displayDate' have different month+year?

Reply:
Date would be the detailed post date with time so that it could be sorted in descending order. Display date is what would be rendered onto the page. I'm thinking it'd be better to have the display date already adjusted for the client, but what do you think?